org.mixedrealitytoolkit.input/InteractionModes/FlatScreenModeDetector.cs

```diff
@@ -34,7 +41,29 @@ protected void Awake()
         public bool IsModeDetected()
         {
             // Flat screen mode is only active if the Left and Right Hand Controllers aren't being tracked
-            return !controllerLookup.LeftHandController.currentControllerState.inputTrackingState.HasPositionAndRotation() && !controllerLookup.RightHandController.currentControllerState.inputTrackingState.HasPositionAndRotation();
+            #pragma warning disable CS0618 // Type or member is obsolete
+            if (controllerLookup != null) //Note: Will be removed for XRI3 migration completion
```

Comment:
Remove `//Note: Will be removed for XRI3 migration completion`

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/f0cb2196739a51e664622c588a6771d0abd87940 , resolving.